### ISSUE TITLE
Tables: Ensure data table headers cells are not blank

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2534]
**OCIO Violation:** Ensure data table headers cells are not blank

**OCIO Notes:** This th element should not be empty

**OCIO priority:** 1

More info in [this sheet](https://doimspp.sharepoint.com/:x:/s/NRRD/Ea-CVbPsGDpDkDLSIe6kZS8BNeTJ7C8_M7yRS86t6VEU2g?e=IXABwa) or the [ocio/amp dashboard](https://departmentofinterior.levelaccess.us/login.php)

**Instances**
- [ ] https://revenuedata.doi.gov/query-data/?dataType=Disbursements
- [ ] https://revenuedata.doi.gov/query-data/?dataType=Federal%20revenue%20by%20company
- [ ] https://revenuedata.doi.gov/how-revenue-works/native-american-revenue/
- [ ] https://revenuedata.doi.gov/how-revenue-works/revenues

**Comment by eelzi-ONRR** [comment]
2024 report

Instances: 218
Occurrence: 13%
Severity: 4
Priority: 1

**Comment by lpgoldstein** [comment]
No longer relevant
